integration_tests/integration/temporaryAbsences.spec.ts

```diff
@@ -0,0 +1,20 @@
+import TemporaryAbsencesPage from '../pages/temporaryAbsences'
+import Page from '../pages/page'
+
+context('SignIn', () => {
+  beforeEach(() => {
+    cy.task('reset')
+    cy.task('stubSignIn')
+    cy.task('stubAuthUser')
+    cy.task('stubIncomingMovements', 'MDI')
+    cy.task('stubTemporaryAbsences', 'MDI')
+  })
+
+  it('A user can view list of temporary absences', () => {
+    cy.signIn()
+    cy.visit('/confirm-arrival/return-from-temporary-absence')
```

Comment:
We could push this url into the TemporaryAbsencesPage and have a static method that returns an instance of the page:

```val temporaryAbsencePage = TemporaryAbsencesPage.goTo()

Reply:
I was struggling to get the right return type initially but think I've managed this now with a return type of Cypress.Chainable<Cypress.AUTWindow> on the goTo() static method